core/src/main/resources/error/error-classes.json

```diff
@@ -757,6 +757,16 @@
       "The expression <sqlExpr> cannot be used as a grouping expression because its data type <dataType> is not an orderable data type."
     ]
   },
+  "HLL_INVALID_LOG_K" : {
+    "message" : [
+      "Invalid call to <function>; the log K value must be between <min> and <max>, inclusive: <value>."
```

Comment:
Similarly: "log" -> "lg" or even "lgK" (one word, no space).

Reply:
The other option is to use the longer name that Ryan has used in the code: "lgConfigK".

Reply:
Done, I went with `lgConfigK`.